lua/telescope/actions/init.lua

```diff
@@ -226,7 +226,7 @@ end
 actions.run_builtin = function(prompt_bufnr)
   local entry = actions.get_selected_entry(prompt_bufnr)
 
-  do_close(prompt_bufnr, true)
+  _do_close(prompt_bufnr, true)
```

Comment:
You have to make the `local do_close` public like this: `actions._do_close` and then update the current impl. But you can wait a little bit. Someone else was also doing this, to fix a bug within `buitlin.git_status`